exam_marker/exam_marker.go

```diff
@@ -124,9 +127,29 @@ func applyWatermark(input io.ReadSeeker, output io.Writer, textLeft string, text
 	conf := pdfcpu.NewDefaultConfiguration()
 	imp := pdfcpu.DefaultImportConfig()
 	imp.Gray = false
-	pdfcpu_api.ImportImages(nil, output, pagesbuf, imp, conf)
-	//io.Copy(output, &tempout)
-
+	buf2 := new(bytes.Buffer)
+	pdfcpu_api.ImportImages(nil, buf2, pagesbuf, imp, conf)
+	ctx, err := pdfcpu.Read(bytes.NewReader(buf2.Bytes()), pdfcpu.NewDefaultConfiguration())
```

Comment:
![image](https://user-images.githubusercontent.com/26827864/152227380-14d24e05-9912-4ed9-b5fc-5c80031fecd9.png)

Reply:
Please also check that error!